src/plugin/cesium/cesiumrenderer.js

```diff
@@ -231,13 +231,17 @@ plugin.cesium.CesiumRenderer.prototype.getCoordinateFromPixel = function(pixel)
     var cartesian = Cesium.Cartesian2.fromArray(pixel);
     var scene = this.olCesium_.getCesiumScene();
     if (scene && scene.camera && scene.globe) {
-      var pickRay = scene.camera.getPickRay(cartesian);
-      cartesian = scene.globe.pick(pickRay, scene);
+      // The Cesium team recommends different methods here based on whether terrain is enabled
+      // see https://github.com/AnalyticalGraphicsInc/cesium/issues/4368#issuecomment-406639086
+      if (scene.terrainProvider) {
```

Comment:
`Cesium.EllipsoidTerrainProvider` is used when terrain is disabled, so this condition will always be true.